### ISSUE TITLE
Are the results in the paper validated sets?

### ISSUE THREAD

**Issue by Renalyn-pan** [514flowey/JDet-cobb#3]
I admire your work very much！But I have a question.The results of the other models in this paper are much lower than those in the corresponding papers, are they the test results on the validation set? and the results of map75 can not be available in the Dota official website evaluation, right?

**Comment by 514flowey** [comment]
We reimplemented all models in this paper. Differences in experimental settings (hyperparameters, data augmentation, etc.) may lead to different results.
The results of mAP75 in the paper are also from DOTA official website.
